tests/functional/Teradata/ExportTest.php

```diff
@@ -244,6 +244,65 @@ public function testExportSimple(): void
         $this->assertContains($this->getExportDir() . '/ts_test/ts_testmanifest', array_values($files));
     }
 
+    public function testExportSimpleAbs(): void
```

Comment:
proc to potrebujeme, v cem se to lisi?

Reply:
ted uz to nepotrebujeme, to byl prvni pokus pro Export na ABS (jeste bez importu)

Reply:
odstraneno v commitu [remove temporary export test](https://github.com/keboola/php-db-import-export/pull/165/commits/24e22d032f02bf854c5b42d6a0e21a562c5bf0f9)

Reply:
ok, smazu to